### PR TITLE
use test harness in tests

### PR DESCRIPTION
**Pull request by vladimir32pa**

### CODE REVIEW

File: app/dashboard/ui.test.js [hunk 1]
```diff
@@ -263,7 +271,7 @@ test('clicking on the client application type section link should navigate the u
 
 // TODO uncomment this test when contact-details page is complete
 // test('clicking on the contact details section link should navigate the user to contact details page', async (t) => {
-//   pageSetup(t);
+//   await pageSetup(t);
```
Comment (sam-roberts-bjss) [inline app/dashboard/ui.test.js]: I assume this test can now be active. Can be changed in another PR.

Comment (vladimir32pa) [reply app/dashboard/ui.test.js]: ok
